src/util/MongoBinaryDownloadUrl.js

```diff
@@ -200,6 +200,8 @@ export default class MongoBinaryDownloadUrl {
       name += '1604';
     } else if (majorVer === 16) {
       name += '1604';
+    } else if (majorVer === 18) {
```

Comment:
Maybe we should add a unit test for this case ?

Reply:
I think we need to merge a fix and publish package ASAP
since it just don't work in Ubuntu 18.04 with default settings.

May be we do not need hard-coded list of versions, LTS Ubuntu versions are always have the same names and mongo binaries files too, so we can just generate right file name for any future releases. 
In this case yes, it will make a sence to add some tests on this function.
But IMHO this will be separate task, more like future proof improvement, not a bug fix.

Reply:
Well I mean there are already of lot of unit tests of this function in the MongoBinaryDownloadUrl-test.js file, I don't think it will be very long to add the specific case for this version, it would be something like this : 
```javascript
it('should return a archive name for Ubuntu 12.04', () => {
      expect(
        downloadUrl.getUbuntuVersionString({
          dist: 'Ubuntu Linux',
          release: '18',
        })
      ).toBe('ubuntu1804');
```

Reply:
As @nodkz asked, I [removed checking `release` field](https://github.com/nodkz/mongodb-memory-server/pull/96/commits/eb3c36e1ec408a624cad7fc86bf821a02915182a),
so this test will be failed instantly)

Reply:
@YuriGor why would it fail ? 
majoVer is constructed by splitting os.release by commas and gathering the first entry : 
```javascript
const ubuntuVer: string[] = os.release ? os.release.split('.') : [];
const majorVer: number = parseInt(ubuntuVer[0], 10);
```
So if you pass this object to the function :
```javascript
{
          dist: 'Ubuntu Linux',
          release: '18',
}
```
majorVer will be equal to 18 so the result will be ubuntu1804 ...

Reply:
ah, majorVer is parsed from release string, ok, test added.